features/features.go

```diff
@@ -589,4 +589,25 @@ var (
 						productScope(ocpSpecific).
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
+
+	FeatureGateUserNamespacesSupport = newFeatureGate("UserNamespacesSupport").
+						reportProblemsToJiraComponent("Node").
+						contactPerson("haircommander").
+						productScope(kubernetes).
+						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
+						mustRegister()
+
+	FeatureGateUserNamespacesPodSecurityStandards = newFeatureGate("UserNamespacesPodSecurityStandards").
+							reportProblemsToJiraComponent("Node").
```

Comment:
This could have gone to `authentication` as well, as it impacts PSa/SCCs; however since PSa/SCC and/or userns expertise might be required for any reported problems, I'm happy to leave this to `Node` in order to keep this coupled with the main FG `UserNamespacesSupport`.